libbeat/docs/field-name-changes.asciidoc

```diff
@@ -12,7 +13,6 @@
 |`docker.container.image`            |`container.image.name`
 |`docker.container.labels`            |`container.labels`
 |`docker.container.name`            |`container.name`
-|`event.type`            |`auditd.message_type`
```

Comment:
This needs to be put back as it was. `event.type` is reserved and should not be used in 7.0